params.go

```diff
@@ -153,6 +153,94 @@ var (
 		1,
 		"Exponential rate param for disaster magnitude",
 	)
+
+	// config.IIGOConfig - Executive branch
+	getRuleForSpeakerActionCost = flag.Float64(
```

Comment:
can we prepend each of these flags with `iigo` so we know these are iigo stuff?

Reply:
Done